Package.swift

```diff
@@ -96,21 +96,21 @@ if let pod_archive_path = ProcessInfo.processInfo.environment["ORT_IOS_POD_LOCAL
     package.targets.append(Target.binaryTarget(name: "onnxruntime", path: pod_archive_path))
 
 } else {
-    // ORT 1.17.0 release
+    // ORT 1.18.0 release
     package.targets.append(
        Target.binaryTarget(name: "onnxruntime",
-                           url: "https://onnxruntimepackages.z14.web.core.windows.net/pod-archive-onnxruntime-c-1.17.0.zip",
-                           checksum: "1623e1150507d9e50554e3d3e5cf9abf75e1bfd8324b74a602acfe45343db871")
+                           url: "https://onnxruntimepackages.z14.web.core.windows.net/pod-archive-onnxruntime-c-1.18.0.zip",
```

Comment:
should we use the new download URL?
```suggestion
                           url: "https://download.onnxruntime.ai/pod-archive-onnxruntime-c-1.18.0.zip",
```

Reply:
ok - did we update the URL for extensions pod as well? (I assume so tried with the new link seems ok to get it downloaded.

Reply:
yes, thanks